wordpress-mobile/gutenberg-mobile
# Disable NPM version 7

`gutenberg` PR: https://github.com/WordPress/gutenberg/pull/30189

The [Gutenberg project doesn't support NPM version 7](https://github.com/WordPress/gutenberg/blob/229aaea79a6131befb3ea9764556b86dd2962b8e/package.json#L19) so we should disable this version in the Gutenberg-mobile too.

Two changes have been applied in this PR:
1. Match npm engine value in the `package.json` file with Gutenberg's project one.
2. Update the install node step in the CircleCI workflow so it doesn't enforce to install the latest NPM version.

To test:
1. Verify that the PR checks pass.
2. Verify that after doing the set up section in the README file, the NPM version installed is below v7.

PR submission checklist:
